RELEASE/scripts/autoscend.ash

```diff
@@ -2924,6 +2924,12 @@ boolean adventureFailureHandler()
 			}
 		}
 
+		if (tooManyAdventures && auto_my_path() == "G-Lover") {
+			if ($locations[The Penultimate Fantasy Airship, The Smut Orc Logging Camp, The Hidden Temple] contains my_location()) {
```

Comment:
Low +item and almost no -combat means you'll be spending a lot of time in these zones.